src/register/RegistrationPage.jsx

```diff
@@ -387,6 +389,7 @@ const RegistrationPage = (props) => {
   const handleOnFocus = (event) => {
     const { name, value } = event.target;
     setErrors(prevErrors => ({ ...prevErrors, [name]: '' }));
+    clearBackendError(name);
```

Comment:
We need to fire this action on `onChange` event when our validation API return 403.
And this should be fire for all those fields for those we are enabling backend validations like email, username, and password.

Reply:
@mubbsharanwar I have discussed this with Sajad and he told me the exact error and when it comes so instead of onchange method it should be called on onfocus. we can discuss it on call